README.md

```diff
@@ -46,6 +46,7 @@ Here's how the default settings look like:
     static_url_base: 'https://ucarecdn.com',
     api_version: '0.5',
     cache_files: true,
+    store_files_upon_uploading: false,
```

Comment:
so let's fix world by one varname at a time.
"autostore" seems to be better, although not 100% correct.

if there are not ideas, I'd go with it.